code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -695,4 +695,7 @@
 	description = "Fluorosulfuric acid is a an extremely corrosive chemical substance."
 	color = "#8E18A9" // rgb: 142, 24, 169
 	toxpwr = 2
-	acidpwr = 20
+	acidpwr = 42.0
```

Comment:
it'll probably resemble how it worked after I first refactored acid, that gif of an assistant throwing one acid grenade at a Sec officer and seeing the officer instantly be surrounded by grey goo.